[BUG] Default setting (`enable`) of `empty-values` & `octal-values` does nothing

There appears to be a contradiction between the meaning of `enable` in the default settings and the actual end result.

#### `empty-values`

https://github.com/adrienverge/yamllint/blob/b62b424dd43acd4b0e08f46f49d88bc15461384e/yamllint/conf/default.yaml#L21

https://github.com/adrienverge/yamllint/blob/0f073f7a09d29ceb85a1aaf388bdbe5f4145f869/yamllint/rules/empty_values.py#L78-L79

#### `octal-values`

https://github.com/adrienverge/yamllint/blob/b62b424dd43acd4b0e08f46f49d88bc15461384e/yamllint/conf/default.yaml#L29

https://github.com/adrienverge/yamllint/blob/0f073f7a09d29ceb85a1aaf388bdbe5f4145f869/yamllint/rules/octal_values.py#L69-L70

---

In both cases, the only way to get these working is to specify them in the `.yamllint` file:

```yaml
rules:
  empty-values:
    forbid-in-block-mappings: true
    forbid-in-flow-mappings: true
  octal-values:
    forbid-implicit-octal: true
    forbid-explicit-octal: true
```

What is `enable` supposed to do?  Otherwise, isn't the default actually `disable` in both cases, with `default.yaml` needing updating (with the knock-on affect to the documentation)?

@adrienverge I think this is going to need a little more thought:

```python
$ python -m unittest -v tests.rules.test_octal_values
test_disabled (tests.rules.test_octal_values.OctalValuesTestCase) ... ok
test_explicit_octal_values (tests.rules.test_octal_values.OctalValuesTestCase) ... FAIL
test_implicit_octal_values (tests.rules.test_octal_values.OctalValuesTestCase) ... FAIL

======================================================================
FAIL: test_explicit_octal_values (tests.rules.test_octal_values.OctalValuesTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/scratch/GitHub/yamllint/tests/rules/test_octal_values.py", line 103, in test_explicit_octal_values
    self.check('implicit-octal: 010', conf)
  File "/home/scratch/GitHub/yamllint/tests/common.py", line 53, in check
    self.assertEqual(real_problems, expected_problems)
AssertionError: Lists differ: [1:20: forbidden implicit octal value "010" (octal-values)] != []

First list contains 1 additional elements.
First extra element 0:
1:20: forbidden implicit octal value "010" (octal-values)

- [1:20: forbidden implicit octal value "010" (octal-values)]
+ []

======================================================================
FAIL: test_implicit_octal_values (tests.rules.test_octal_values.OctalValuesTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/scratch/GitHub/yamllint/tests/rules/test_octal_values.py", line 82, in test_implicit_octal_values
    self.check('explicit-octal: 0o10', conf)
  File "/home/scratch/GitHub/yamllint/tests/common.py", line 53, in check
    self.assertEqual(real_problems, expected_problems)
AssertionError: Lists differ: [1:21: forbidden explicit octal value "0o10" (octal-values)] != []

First list contains 1 additional elements.
First extra element 0:
1:21: forbidden explicit octal value "0o10" (octal-values)

- [1:21: forbidden explicit octal value "0o10" (octal-values)]
+ []

----------------------------------------------------------------------
Ran 3 tests in 0.162s

FAILED (failures=2)
```

--- a/tests/rules/test_octal_values.py
+++ b/tests/rules/test_octal_values.py
@@ -28,7 +28,9 @@ class OctalValuesTestCase(RuleTestCase):
         self.check('user-city: 0o10', conf)
 
     def test_implicit_octal_values(self):
-        conf = ('octal-values: {forbid-implicit-octal: true}\n'
+        conf = ('octal-values:\n'
+                '  forbid-implicit-octal: true\n'
+                '  forbid-explicit-octal: false\n'
                 'new-line-at-end-of-file: disable\n'
                 'document-start: disable\n')
         self.check('user-city: 010', conf, problem=(1, 15))
@@ -50,7 +52,9 @@ class OctalValuesTestCase(RuleTestCase):
                    '  - 0e3\n', conf)
 
     def test_explicit_octal_values(self):
-        conf = ('octal-values: {forbid-explicit-octal: true}\n'
+        conf = ('octal-values:\n'
+                '  forbid-implicit-octal: false\n'
+                '  forbid-explicit-octal: true\n'
                 'new-line-at-end-of-file: disable\n'
                 'document-start: disable\n')
         self.check('user-city: 0o10', conf, problem=(1, 16))

--- a/yamllint/conf/default.yaml
+++ b/yamllint/conf/default.yaml
@@ -18,7 +18,7 @@ rules:
   document-start:
     level: warning
   empty-lines: enable
-  empty-values: enable
+  empty-values: disable
   hyphens: enable
   indentation: enable
   key-duplicates: enable
@@ -26,7 +26,7 @@ rules:
   line-length: enable
   new-line-at-end-of-file: enable
   new-lines: enable
-  octal-values: enable
+  octal-values: disable
   quoted-strings: disable
   trailing-spaces: enable
   truthy:

--- a/yamllint/rules/empty_values.py
+++ b/yamllint/rules/empty_values.py
@@ -75,8 +75,8 @@ ID = 'empty-values'
 TYPE = 'token'
 CONF = {'forbid-in-block-mappings': bool,
         'forbid-in-flow-mappings': bool}
-DEFAULT = {'forbid-in-block-mappings': False,
-           'forbid-in-flow-mappings': False}
+DEFAULT = {'forbid-in-block-mappings': True,
+           'forbid-in-flow-mappings': True}
 
 
 def check(conf, token, prev, next, nextnext, context):

--- a/yamllint/rules/octal_values.py
+++ b/yamllint/rules/octal_values.py
@@ -66,8 +66,8 @@ ID = 'octal-values'
 TYPE = 'token'
 CONF = {'forbid-implicit-octal': bool,
         'forbid-explicit-octal': bool}
-DEFAULT = {'forbid-implicit-octal': False,
-           'forbid-explicit-octal': False}
+DEFAULT = {'forbid-implicit-octal': True,
+           'forbid-explicit-octal': True}
 
 
 def check(conf, token, prev, next, nextnext, context):

